mkt/reviewers/tests/test_views.py

```diff
@@ -4146,3 +4146,60 @@ def test_comm_url_no_encode(self):
         eq_(doc('#history .item-history').attr('data-comm-thread-url'),
             reverse('comm-thread-list') + '?limit=1&app=' +
             self.addon.app_slug)
+
+
+class TestModerateLog(AppReviewerTest):
+
+    def setUp(self):
+        super(TestModerateLog, self).setUp()
+        self.login_as_editor()
+        self.url = reverse('reviewers.apps.moderatelog')
+        self.review = Review.objects.create(addon=app_factory(), body='body',
+                                            user=user_factory(), rating=4,
+                                            editorreview=True)
+        mkt.set_user(UserProfile.objects.get(username='editor'))
+
+    def test_log(self):
+        r = self.client.get(self.url)
+        eq_(r.status_code, 200)
+
+    def test_start_filter(self):
+        r = self.client.get(self.url, dict(start='2011-01-01'))
+        eq_(r.status_code, 200)
```

Comment:
Yeah.  There are tests for start being provided; end being provided; and the filter.  Maybe a bit simplistic I suppose.